configuration.py

```diff
@@ -19,4 +18,13 @@ class Configuration(Enum):
     BOT_ADDRESS = 'localhost'
 
     # -------- MONGO COLLECTIONS NAMES -----------#
+    # note: each enum name which refers to collections should contain
+    # 'collection' key word - it allows for automatic collections creation, another key word is 'capped'
```

Comment:
what is 'capped' key word use for?

Reply:
it is for collections which behave like cyclic buffer, or collections with a size constraints.